tables/hu.tbl

```diff
@@ -1,22 +1,24 @@
+# liblouis: Hungarian Grade 1 table
+#
+# -----------
 #-index-name: Hungarian, partially contracted
 #-display-name: Hungarian partially contracted braille
-
+#
 #+locale:hu
 #+type:literary
 #+contraction:partial
 #+grade:1
 # Marked as "direction:both" by Bue Vester-Andersen
 # as tests run both forward and backward
 #+direction:both
-
+#+hyphenation:yes
 # TODO: Please correct the metadata above. It is not meant to be
 # accurate nor complete. It hasn't been verified by the table
 # author yet. It is merely an attempt by the liblouis maintainers
 # to get some sensible initial values in place.
-
 # TODO: Please add a reference to official documentation about
 # the implemented braille code. Preferably submit the documents
 # to https://github.com/liblouis/braille-specs.
-
 include hu-hu-g1.ctb
+include hyph_hu_HU.dic
```

Comment:
I don't like this. We want to get rid of all hyphenation tables in Liblouis (except when they are needed for `nocross` rules, like in the Danish tables).